holidays/countries/lebanon.py

```
@@ -0,0 +1,438 @@
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars import _CustomIslamicHolidays
+from holidays.calendars import _CustomIslamicHolidays
```

Ruff (0.11.9)

Redefinition of unused `_CustomIslamicHolidays` from line 15

Remove definition: `_CustomIslamicHolidays`

(F811)